python/src/puml/gen_puml.aac

```diff
@@ -0,0 +1,131 @@
+              - A PlantUML sequence diagram of the use case is written to the output directory.
+          - name: Output PlantUML sequence diagram to file.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid filepath.
+            when:
+              - The aac app is run with the puml-sequence command.
+            then:
+              - A PlantUML sequence diagram of the model is generated to a .puml file.
+    - name: puml-object
+      help_text: Convert an AaC-defined system to PlantUML object diagram.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML object diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Object Diagram Test
+        scenarios:
+          - name: Output PlantUML object diagram for valid model.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+            when:
+              - The aac app is run with the puml-object command.
+            then:
+              - A PlantUML object diagram of the model is written to the output directory.
+          - name: Output PlantUML object diagram to file.
+            given:
+              - The {{puml.input.architecture-file}} contains a valid model.
+              - The positional argument {{puml.input.output-directory}} contains a valid filepath.
+            when:
+              - The aac app is run with the puml-object command.
+            then:
+              - A PlantUML object diagram of the model is generated to a .puml file.
+    - name: puml-requirements
+      help_text: Convert an AaC-defined system to a requirements diagram in PlantUML format.
+      input:
+        - name: architecture-file
+          type: file
+          description: |
+            A path to a YAML file containing an AaC-defined system from which to
+            generate a PlantUML requirements diagram.
+        - name: output-directory
+          type: directory
+          description: |
+            The output directory into which the PlantUML (.puml) diagram file
+            will be written.
+      acceptance:
+        name: Requirement Diagram Test
+        scenarios:
+        - name: Output a requirements diagram in PlantUML format with a valid system model.
+          given:
+            - The {{puml-requirement.input.architecture-file}} contains a valid model.
+          when:
+            - The aac app is run with the puml-requirement command.
+          then:
+            - A PlantUML object diagram of the model is written to the output directory.
```

Comment:
If you are wanting to repurpose this scenario, then it needs more than just the result modified. As it is right now, it makes it seem that you can provide only the architecture file and get a result, and that is incorrect. 

```suggestion
          - name: Output PlantUML requirements diagram to the requested output directory.
            given:
              - The {{puml-requirement.input.architecture-file}} contains a valid model.
              - The positional argument {{puml-requirement.input.output-directory}} contains a valid file path.
            when:
              - The aac app is run with the puml-component command.
            then:
              - A PlantUML requirements diagram of the model is written to the output directory.
```